fivetran_blueprints/check_sync_status.py

```diff
@@ -138,14 +136,13 @@ def main():
             if full_pickle_path:
                 connector_id, execution_time = load_pickle_variables(
                     full_pickle_path)
-
+    elif args.connector_id:
```

Comment:
@wrp801 Might be a dumb question, but `elif` only checks if the `if` isn't met right? So currently this still doesn't solve for the case of:
- Use the user-provided connector id even if there's an upstream Vessel.

It currently reads like it would ignore what the user provided as the connector ID if something is run upstream.